docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -277,6 +277,10 @@ AgroalDataSource dataSource2;
 |io.quarkus.agroal.runtime.TransactionIsolationLevel (enum)
 |none, read-committed, read-uncommitted, repeatable-read, serializable
 
+|quarkus.datasource.enable-metrics
+|Boolean
+|True or False
```

Comment:
Mmh, I think we should make it `true` or `false` all lowercase.

Same for the other booleans.

Reply:
I'll make the other modification in this same commit.